Prace_domowe/Praca_domowa1/Grupa1/Adam_Frej/PD1.ipynb

```diff
@@ -0,0 +1,1134 @@
+{
```

Comment:
<p>Wykres zmiennej area to klasyczny przykład rozkładu z grubym ogonem.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>